frontend/src/components/where-to-buy/where-buy-item/styles.module.scss

```diff
@@ -0,0 +1,239 @@
+@import "@assets/css/variables";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  padding: 15px 25px;
+  background-color: var(--white);
+  border: solid 1px var(--white);
+  border-radius: 5px;
+
+  .logo {
+    display: flex;
+    align-items: center;
+    width: 93px;
+    margin-right: 15px;
+    padding: 0 3px 0 0;
+
+    img {
+      width: 90px;
+      margin: 0;
+    }
+  }
+
+  .description {
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    width: auto;
+
+    .carName {
+      margin-bottom: 5px;
+      font-weight: 600;
+      font-size: 1.3rem;
+      text-transform: uppercase;
+    }
+
+    .labelGroup {
+      display: flex;
+      margin-bottom: 5px;
+
+      & > div {
+        margin-right: 1rem;
+      }
+
+      & > div:last-child {
+        margin-right: 0;
+      }
+    }
+
+    .label {
+      display: flex;
+      align-items: center;
+      height: 2.5rem;
+      padding: 5px 10px;
+      color: var(--white);
+      font-weight: 600;
+      font-size: 1rem;
+      font-family: var(--primary-font);
+      font-style: normal;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      background: var(--blue300);
+      border-radius: 4px;
+
+      & > img {
+        height: 1rem;
+        margin-right: 0.5rem;
+      }
+
+      &:last-child {
+        margin-bottom: 0;
+      }
+
+      &:hover {
+        cursor: pointer;
+      }
+    }
+
+    .label.location {
+      text-transform: capitalize;
+    }
+
+    .trigger {
+      position: relative;
+      display: flex;
+      padding: 0;
+      font-size: 1.2rem;
+
+      .textDescription {
+        /* stylelint-disable-next-line value-no-vendor-prefix */
+        display: -webkit-box;
+        -webkit-line-clamp: 3;
+        -webkit-box-orient: vertical;
+        margin-bottom: 1.5rem;
+        padding: 0 10px;
+        overflow: hidden;
+      }
+
+      .textDescription.more {
+        display: inline-block;
+      }
+
+      div::before {
+        content: "\2022";
```

Comment:
We would always have 2022? 
Still in 2025?

Reply:
it is not a year, it is code of
https://gyazo.com/985212dd79efca44004f2e0e3c0b4975